src/org/zalando/stups/friboo/system/metrics.clj

```diff
@@ -51,22 +39,22 @@
                                 "*ROOT*"
                                 path))]))
 
-(defn collect-swagger1st-zmon-metrics
+(defn collect-swagger1st-request-metrics
   "Ring middleware that creates Timers for Swagger API calls
   and stores them in the running metrics component."
-  [next-handler component]
-  (if-let [metrics-registry (:metrics-registry component)]
-    ; then
+  [next-handler {:keys [metrics-registry configuration]}]
+  (if metrics-registry
     (fn [request]
-      (let [start (System/currentTimeMillis)
+      (let [start    (System/currentTimeMillis)
             response (next-handler request)
-            status (:status response)
-            timer (tmr/timer metrics-registry ["zmon" "response" (swagger1st-request2string request status)])]
+            status   (:status response)
+            ;; TODO replace "zmon" with configuration value
```

Comment:
Yes, it's done.
By default it's "friboo", see `default-configuration` on line 100.
For internal Zalando projects "zmon" will be set: https://github.com/zalando-incubator/friboo-ext-zalando/blob/master/src/org/zalando/stups/friboo/zalando_internal/config.clj#L10